### PR TITLE
feat(id): add option to change ingress-descriptor filename

### PR DESCRIPTION
**Pull request by youen**

### CODE REVIEW

File: internal/app/pull/http.go [hunk 1]
```diff
@@ -27,107 +27,109 @@ import (
...
-				_, ew := w.Write([]byte("{\"error\": \"param filter must be a string map (key1:value1,key2:value2)\"}\n"))
+				_, ew := w.Write([]byte("{\"error\" : \"param limit must be an positive integer\"}\n"))
 				if ew != nil {
 					log.Error().Msg("Write failed")
 					return
 				}
 				return
 			}
-			filter[kv[0]] = kv[1]
+			limit = uint(limit64)
```
Comment (youen) [inline internal/app/pull/http.go:77]:
```suggestion
			limit = uint(limit64)
```

File: internal/app/pull/http.go [hunk 2]
```diff
@@ -27,107 +27,109 @@ import (
...
+						return
+					}
+					return
+				}
+				filter[kv[0]] = kv[1]
+			}
+		}
+
+		if query.Get("limit") != "" {
+			limit64, elimit := strconv.ParseUint(query.Get("limit"), 10, 64)
```
Comment (youen) [inline internal/app/pull/http.go:66]:
```suggestion
			limit64, elimit := strconv.ParseUint(query.Get("limit"), 10, 32)
```

File: README.md [hunk 3]
```diff
@@ -135,6 +135,11 @@ IngressDescriptor:
             name: public.language
 ```
 
+### `--ingress-descriptor` argument
```
Comment (CapKicklee) [inline README.md:138]: I think there could be a reference to the short version -i here

**Review by CapKicklee — COMMENTED** [review]
Apart from the unint cast, I've nothing to say, it looks good to me!

File: README.md (hunk 3 above)
Comment (adrienaury) [reply README.md:138]: > Apart from the doc, I've nothing to say, it looks good to me!

And the CHANGELOG :)

**Review by adrienaury — APPROVED** [review]

**Pull request merged**